content/docs/service-level-objectives.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Service Level Objectives
+date: 2022-03-31
+weight: 10
+---
+
+# Service Level Objectives
+
+Packit is using the concept of [Service level
+objectives](https://sre.google/sre-book/service-level-objectives/) defined in
+Google's SRE book. It means that when we run out of error budget, we stop all
+the feature work and work on stabilizing the service to get our budget back.
```

Comment:
```suggestion
the work on new features and start stabilizing the service to get our budget back.
```
it just sounds a little better to me, so only a suggestion

Reply:
yup, I just realized it today when reading it again, thanks!